### DISCUSSION TITLE
Building blogs with Netlify CMS

### DISCUSSION THREAD

**Discussion by wutali** [vercel/next.js#16340] [Show and tell]
Recently I tried to build a website with Next.js and Netlify stacks. I want some standard blogging features for my project, but there is no good example for realizing them. The template I built has the following features:

- **Tagging**: organizes content by tags
- **Author**: displays author names who write a post
- **Pagination**: limits the number of posts per page
- **CMS**: built with CMS to allow editors modifying content with the quickest way
- **SEO optimized**: built-in metadata like JSON-LD
- **Shortcode**: extends content writing with React component like WordPress shortcodes

I believe it is enough for implementing a website like blogs. 

Let me know if you found what I need to fix, or something was missing for building your website with this template.

**Comment by Bowens20832** [comment]
@wutali Have you found any updates to this dicussion?

I think I have arrived at the same point while building a free open-source content creation platform on top of Next.js and Vercel: [Elegant](https://github.com/elegantframework/elegant-cli).

No big deal, we will work a solution out 😄  

We just don't want to reinvent the wheel if we don't have to.
